src/app.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable quotes */
```

Comment:
- [ ] Currently, all your code for the app is in this file. You are required to separate all the application logic from the DOM-related stuff. Add new files for creating a new todo, another for creating a new project, another for the DOM manipulation. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/modules_requirement.png) from Odin)